int-set/src/lib.rs

```diff
@@ -0,0 +1,165 @@
+mod bitpage;
+mod bitset;
+
+use bitset::BitSet;
+use std::ops::RangeInclusive;
+
+/// A fast & efficient unsigned integer (u32) bit set which is invertible.
+#[derive(Clone, Debug)]
+pub enum IntSet<T> {
+    /// Records a set of integers which are members of the set.
+    Standard(BitSet<T>),
+
+    /// Records the set of integers which are not members of the set.
+    Inverted(BitSet<T>),
+}
+
+impl<T: Into<u32> + Copy + Default> IntSet<T> {
+    /// Create a new empty set.
+    pub fn default() -> IntSet<T> {
```

Comment:
we should implement `default` through the `Default` trait, and here we should just have a `new` constructor.